jobs/build.Jenkinsfile

```diff
@@ -75,7 +75,7 @@ def strict_build_param = stream_info.type == "mechanical" ? "" : "--strict"
 // here; we can look into making them configurable through the template if
 // developers really need to tweak them (note that in the default minimal devel
 // workflow, only the qemu image is built).
-def cosa_memory_request_mb = 6.5 * 1024 as Integer
+def cosa_memory_request_mb = 10 * 1024 as Integer
```

Comment:
Hmm, but this will increase the number of parallel kola tests. I think we want instead to flip it around so that we derive the memory request from `ncpus`? E.g.

```groovy
def ncpus = 6
def cosa_memory_request_mb = ((ncpus * 1024) + X) as Integer
```

And then solve for X since it sounds like 512M is actually not enough. :)

Reply:
the kola tests aren't actually the problem. it's the `testiso` tests which don't don't have a direct relationship with this memory request IIUC.

Reply:
https://github.com/coreos/fedora-coreos-pipeline/pull/688